updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -6,20 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-[id="update-restricted-network-cluster-update-service"]
```

Comment:
It looks like we've had this anchor since 4ec62ac9166f74 (#41224).  I'm not  actually clear where this shows up today, but is it an anchor we need to preserve for backwards compatibility with existing user bookmarks or other incoming links?

Reply:
Normally it's good practice to keep anchors as they are to preserve those links, yes. 

However, in this case I couldn't find any way on the doc site for a user to actually get a URL with this specific anchor in it. When you click on the anchor icon for "About the OpenShift Update Service", the anchor in the URL is actually the one that comes from the module: "update-service-overview_updating-restricted-network-cluster-osus".  

So I don't think it'll break any bookmarks, and I couldn't find any incoming links in our repo that use this anchor.